### PR TITLE
[FEATURE]: Controller to update comment #11

### PR DESCRIPTION
**Pull request by Lihemen**
**What does this PR do?**

Provides a user with the control to update a comment made on a post.

**description of Task to be completed?**

This PR provides a controller and an endpoint for updating a comment. 

**How should this be manually tested?**

- `clone this repository to your system`
- `open folder in an editor; preferably VSCode`
- `open up the terminal`
- `run npm install` to install dependencies
- `run npm startDev` to spin up development server
- using an endpoint tester like `POSTMAN`, send a `PATCH` request to `localhost:4000/comments/:commentId`, where ` :commentId` is the id of a comment

**What is the link to the issue on Github?**

[Issue #11 ](https://github.com/microapidev/comment-microapi/issues/11)

### CODE REVIEW

File: controller/commentsController.js [hunk 1]
```diff
@@ -51,3 +52,37 @@ exports.flagComment = async (req, res) => {
     errHandler(error, res);
   }
 };
+
+exports.updateComment = async (req, res, next) => {
+  const comment_id = req.params.commentId;
+  const commentBody = req.body.commentBody;
+  if (!commentBody || commentBody.trim() == "") {
+    return res.status(400).json({ message: "Invalid comment body" });
```
Comment (simply-alliv) [inline controller/commentsController.js:60]: Please use the responseHandler from the utils folder.

File: controller/commentsController.js [hunk 2]
```diff
@@ -51,3 +52,37 @@ exports.flagComment = async (req, res) => {
...
+  }
+  Comments.findById(comment_id)
+    .exec()
+    .then((comment) => {
+      if (!comment) {
+        return next(new CustomError(404, "Comment not found!!"));
```
Comment (simply-alliv) [inline controller/commentsController.js:66]: Please use responseHandler. The CustomError is for when there is an errror thrown.

File: controller/commentsController.js (hunk 1 above)
Comment (Lihemen) [reply controller/commentsController.js:60]: Okay. I’ll send an update right away

Comment (simply-alliv) [reply controller/commentsController.js:60]: Okay, thanks.

**Comment by oscar-ekeyekwu** [comment]
@Lihemen did you run `npm run test`

**Pull request closed without merging**